script.js

```diff
@@ -86,7 +86,7 @@ const x = {
     modpara1: 'CANCOPY',
     modli1: 'Back End Dev',
     modli2: '2015',
-    modpara2: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem excepturi cupiditate placeat suscipit quia  consequuntur reprehenderit sit fuga officiis cupiditate optio dolore labore vero itaque adipisci nihil ipsam, quisquam, quasi officia dolorem consectetur. Modi, adipisci. Laborum maxime ipsa impedit ut, omnis libero sequi fuga at esse nostrum? Voluptatibus, labore! Corporis quae recusandae, repudiandae !',
+    modpara2: ' This is Cancopy Project',
     modli3: 'html',
     modli4: 'css',
     modli5: 'javascript',
```

Comment:
- [ ] As mentioned by the previous reviewer, the project popup does not appear very well because the `see live` and `see source` button is partially hidden when you view using google chrome browser. See below 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/224814229-dfe6a7c0-e9fb-4551-8023-b7a0e17d0437.png)

- One solution to this is to add a scroll effect to the container or popup wrapper. I recommend you do this as it will allow a user to scroll down to view hidden content irrespective of whatever browser the use. 

- [ ] Secondly, the text below the popup picture has more words as seen below 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/224815985-9730662e-6cbf-4d09-9b43-725db31e8efc.png)

- If you add the scrolling effect as suggested above, your popup can text more words as expected because, user will be able to scroll down to see hidden text/buttons.